site/app/controllers/OfficeHoursQueueController.php

```diff
@@ -452,6 +452,29 @@ public function changeRegex($queue_code) {
         );
     }
 
+    /**
+     * @Route("/courses/{_semester}/{_course}/office_hours_queue/{queue_code}/change_contact_information", methods={"POST"})
+     * @AccessControl(role="LIMITED_ACCESS_GRADER")
+     * @return MultiResponse
+     */
+    public function changeContactInformation($queue_code) {
+        if (empty($queue_code)) {
+            $this->core->addErrorMessage("Missing queue name");
```

Comment:
Is queue code the same thing as queue name or should this message be updated

Reply:
They are the same thing

Reply:
Ok sounds good